server/server.js

```diff
@@ -1,17 +1,46 @@
 const path = require('path');
 const express = require('express');
-const server = express();
+const app = express();
+const DBController = require('./database/DBController');
+
 const publicPath = path.join(__dirname, '..', 'build');
 const port = process.env.PORT || 3000;
 
-server.use(express.static(publicPath));
+const db_controller = new DBController();
+
+// TODO: in the future delete and use only `process.env.MONGODB_URI`
+const MAIN_DB_NAME = process.env.MONGODB_URI ? process.env.MONGODB_URI.slice(process.env.MONGODB_URI.lastIndexOf('/') + 1) : 'heroku_hnd618dr';
+
+db_controller.connect().then(() => {
+
+  app.use(express.static(publicPath));
+
+
+  app.get('/fetchData', (req, res) => {
+    if (!req.query.collectionName) {
```

Comment:
I don't think that collectionName should be part of query params.

Reply:
changed in a new branch